README.md

```diff
@@ -87,16 +87,22 @@ More details in the PRD: https://docs.google.com/document/d/1F6qHZ98POQ-CXGzxLPp
    * String email
    * String password
    * Transaction List transactions
-   * Dictionary spendingLimit
-   * Dictionary amountSpent
+   * Budget spendingLimit
+   * Budget amountSpent
 
 * Transaction
    * String date
    * Float amount
    * String category
    * String description
+   * User user
+
+* Budget
```

Comment:
Here you call it a budget which makes more sense to me than Spending - maybe even a CategoryBudget to be even more clear? since it's not the whole budget, it's just a category of the budget

Reply:
I should update the readme since I made some new classes-- I currently have a Transaction class for purchases that the user tracks, a Budget class for the user's spending limit (there can be budgets for overall spending or specific categories), and a Spending class that's very similar to the Budget class but stores the user's total spendings so far (overall or for each category). Do you think different names would make more sense?